config/schema_by_entity_mapping.json

```diff
@@ -1,6 +0,0 @@
-{
```

Comment:
I think we need this config file.
How would you load a schema without a config when - soon - we have number of different schema for a sample.
I also realised that the config needs to be fixed for this purpose, anyway.